PurchasesCoreSwift/Logging/Strings/ConfigureStrings.swift

```diff
@@ -1,29 +1,37 @@
 //
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
 //  ConfigureStrings.swift
-//  PurchasesCoreSwift
 //
 //  Created by Tina Nguyen on 12/11/20.
-//  Copyright © 2020 Purchases. All rights reserved.
 //
 
 import Foundation
 
 // swiftlint:disable identifier_name
-@objc(RCConfigureStrings) public class ConfigureStrings: NSObject {
+class ConfigureStrings {
```

Comment:
in another PR we could make these all enums

Reply:
#768